packages/@aws-cdk/aws-ec2/lib/connections.ts

```diff
@@ -47,7 +48,7 @@ export interface ConnectionsProps {
    *
    * @default No default port range
    */
-  readonly defaultPortRange?: IPortRange;
+  readonly defaultPortRange?: Port;
```

Comment:
And there here, should the `Port` type be called `PortRange`?

Reply:
Yeah I've decided to pretend that a PortRange is-a Port, as opposed to a Port is-a PortRange, because the common case will turn out shorter and nicer, and I don't think it's all that confusing.